qCC/ccGraphicalSegmentationTool.h

```diff
@@ -19,7 +19,7 @@
 #define CC_GRAPHICAL_SEGMENTATION_TOOLS_HEADER
 
 //Local
-#include <ccOverlayDialog.h>
+#include "ccOverlayDialog.h"
```

Comment:
Please restore the previous syntax, as this header is not next to the ccGraphicalSegmentationTool.h header (quotes are for relative paths)